website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -82,6 +82,16 @@ Before upgrading to Nomad 1.10, perform the following tasks:
 Refer to [Migrating to Using Workload Identity with Vault][] for more
 details.
 
+#### Consul template implicit workload identity removal
+Nomad will no longer create an implicit Consul identity for workloads that don't
```

Comment:
```suggestion
Nomad no longer creates an implicit Consul identity for workloads that don't
```

Not an expert here but I think our guidelines say we should use present tense in phrases like that.